util/ct_log_list.cnf

```diff
@@ -0,0 +1,9 @@
+# This file specifies the Certificate Transparency logs
+# that are to be trusted.
+
+# This file is empty in the OpenSSL source distribution.  One way
+# to generate a file (such as from Chrome's trust list) is the following:
+#       wget https://raw.githubusercontent.com/google/certificate-transparency/master/python/utilities/log_list/print_log_list.py
+#       python print_log_list.py --openssl_output ct_log_list.cnf https://sites.google.com/site/certificatetransparency/known-logs.
```

Comment:
This wouldn't work. You'd have to wget the other Python files in that directory at the minimum. Plus, there are a bunch of other flags required for print_log_list.py to actually work. I think it'd be best to just:

- link to www.certificate-transparency.org/known-logs
- link to https://github.com/google/certificate-transparency/blob/master/python/utilities/log_list/print_log_list.py
- mention that the `--openssl_output` flag is required
- leave it as an exercise for the reader to figure out exactly how they invoke the command

That provides all of the pieces of the puzzle, and then the reader just needs to read `print_log_list.py --help`. It should be the responsibility of the CT project to improve their documentation for how that tool is used, if it isn't clear enough.